app/helpers/value_helper.rb

```diff
@@ -2,6 +2,18 @@
 
 module ValueHelper
   # Renderers
+
+  # Given a fedora object, fetch the preferred object title
+  # @return [String] object title, as pulled from metadata, given the following order of preference:
+  #  1. MODS title, 2. identityMetadata label, 3. DC title, 4. Fedora label
+  def object_title(obj)
+    obj_title = obj.descMetadata.title_info.first.strip if obj.descMetadata.title_info.first.present?
```

Comment:
Can you just return here, rather than doing the `||=` below?

Reply:
yup, something like this?

```    
return obj.descMetadata.title_info.first.strip if obj.descMetadata.title_info.first.present?
obj_title = obj.identityMetadata.objectLabel.first.strip if obj.identityMetadata.objectLabel.first.present?
obj_title ||= obj.DC.title.first.strip if obj.DC.title.first.present?
obj_title ||= obj.label
obj_title```

Reply:
yes, but then also on the next 3 lines too.

Reply:
ahh right, fixed